frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -392,12 +423,49 @@ const SECTIONS = updateSectionsWithPlugins({
         widget: EmbeddedQuestionListing,
         getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
       },
+      {
+        widget: EmbeddingCustomizationInfo,
+        getHidden: (_, derivedSettings) =>
+          !derivedSettings["enable-embedding"] ||
+          MetabaseSettings.isEnterprise(),
+      },
+      {
+        widget: () => (
+          <RedirectWidget to="/admin/settings/embedding_in_other_applications" />
```

Comment:
redirect users to the main setting page when embedding isn't enabled. As it won't make sense to be able to visit the sub-setting page unless the embedding is enabled.